plugins/Goals/tests/System/expected/test_trackGoals_pages__Actions.getPageTitles_week.xml

```diff
@@ -2,134 +2,174 @@
 <result>
 	<row>
 		<label> Page A - index.html</label>
-		<nb_visits>4</nb_visits>
-		<nb_uniq_visitors>4</nb_uniq_visitors>
-		<nb_hits>5</nb_hits>
-		<sum_time_spent>1800</sum_time_spent>
+		<nb_visits>7</nb_visits>
+		<nb_hits>9</nb_hits>
+		<sum_time_spent>2880</sum_time_spent>
 		<sum_bandwidth>0</sum_bandwidth>
 		<nb_hits_with_bandwidth>0</nb_hits_with_bandwidth>
 		<min_bandwidth />
 		<max_bandwidth />
-		<entry_nb_uniq_visitors>4</entry_nb_uniq_visitors>
-		<entry_nb_visits>4</entry_nb_visits>
-		<entry_nb_actions>18</entry_nb_actions>
-		<entry_sum_visit_length>5047</entry_sum_visit_length>
+		<entry_nb_visits>7</entry_nb_visits>
+		<entry_nb_actions>33</entry_nb_actions>
+		<entry_sum_visit_length>9373</entry_sum_visit_length>
 		<entry_bounce_count>0</entry_bounce_count>
 		<goals>
 			<row idgoal='1'>
-				<nb_conversions>3</nb_conversions>
-				<revenue>30</revenue>
-				<nb_conv_pages_before>9</nb_conv_pages_before>
-				<nb_conversions_attrib>1.0833</nb_conversions_attrib>
+				<nb_conversions>5</nb_conversions>
```

Comment:
are these number changes expected?

Reply:
The test dates were changed and some tests changed from day to week periods, so some changes are expected. I've checked the values against manual calculations and they appear to be correct.